src/main/java/kitEditor/Sample.java

```diff
@@ -27,10 +27,27 @@ public Sample(short[] iBuf, String iName) {
         name = iName;
     }
 
+    Sample(Sample s) {
+        file = s.file;
+        name = s.name;
+        originalSamples = s.originalSamples;
+        processedSamples = s.processedSamples;
```

Comment:
I suspect it would be safer to make copies of file, name, originalSamples and processedSamples, rather than just copy the reference.

Probably it doesn't matter though, for the simple copy/paste scenario